File: morning_exercises/day2.cs
public static class Palindrome
{
    public static bool IsPalindrome(int n)
    {
        if (n<0) return false;

        int palCheck = 0, temp = n;

        while (temp>0)
        {
            int lastDigit = temp%10;
            palCheck=palCheck*10+lastDigit;
            temp/=10;
        }

        return palCheck==n;
    }
}

File: morning_exercises/day2.js
function isPalindrome(n) {
    if (typeof(n)!= "number" || !Number.isInteger(n)) throw new Error("Input must be an integer");

    if (n<0) return false;

    let checkPal=0, temp=n;

    while (temp>0) {
        let lastDigit = temp%10;
        checkPal=checkPal*10+lastDigit;
        temp=Math.floor(temp/10);
    }
    return checkPal===n;
}

console.log(isPalindrome(10));